src/printer.js

```diff
@@ -510,6 +510,12 @@ function genericPrintNoParens(path, options, print) {
       if (hasContent) {
         parts.push(indent(options.tabWidth, concat([ hardline, naked ])));
       }
+      else if(n.comments) {
+        parts.push(
+          indent(options.tabWidth,
```

Comment:
the awesome part about prettier is that I don't need to tell you

nit: should be

```js
indent(
  options.tabWidth,
  comments.printDanglingComments(path, print, options))
)
```